packages/cli/src/databases/repositories/usageMetrics.repository.ts

```diff
@@ -60,8 +60,13 @@ export class UsageMetricsRepository extends Repository<UsageMetrics> {
 				(SELECT SUM(count) FROM ${workflowStatsTable} WHERE name IN ('manual_success', 'manual_error')) AS manual_executions_count;
 		`)) as Row[];
 
-		const toNumber = (value: string | number) =>
-			typeof value === 'number' ? value : parseInt(value, 10);
+		const toNumber = (value: string | number) => {
+			const num = typeof value === 'number' ? value : parseInt(value, 10);
+
+			if (isNaN(num)) return 0;
+
+			return num;
+		};
```

Comment:
```suggestion
		const toNumber = (value: string | number) =>
			typeof value === 'number' ? value : parseInt(value, 10) || 0;
```

Reply:
I prefer not to rely on `||` to coerce falsy values. Easy to forget that `NaN` is falsy, and being explicit about `isNaN()` is preferable no?

Reply:
The thing about `||` in general makes sense, but since `parseInt` produces only number or `NaN`, it's fine to just use `|| 0`. 
I think a generally good idea in this case would be to trust the `prefer-nullish-coalescing` rule. If it doesn't complain on a `||`, then it's fine.